### PR TITLE
Extend functional tests for meters

### LINKED ISSUE #1831: Verify that meters are removed when flow is removed
1. Verify for default flow (this may already exist, don't remember)
2. Verify for flow with ignore_bandwidth=true
3. Verify that meters are NOT installed for flows with bandwidth=0

### PR DESCRIPTION
**Pull request by ylobankov**
Closes #1831.

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy [hunk 1]
```diff
@@ -50,8 +52,6 @@ class MetersSpec extends BaseSpecification {
     }
 
     def "Default meters should be re-calculated to kbps on Centec switches"() {
-        requireProfiles("hardware")
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy]: this was an intentional duplication, I asked about it in Nikolay's PR. When L34 gets removed, this should remain hardware-only test.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy [hunk 2]
```diff
@@ -125,6 +126,31 @@ class MetersSpec extends BaseSpecification {
...
+
+    def "Meters are not created when creating a flow with ignore_bandwidth = 0"() {
+        given: "A switch with OpenFlow 1.3 support"
+        def sw = topology.getActiveSwitches().find { it.ofVersion == "OF_13" }
+
+        when: "Get default meters from the switch"
+        def defaultMeters = northbound.getAllMeters(sw.dpId)
+        assert defaultMeters
+
+        and: "Create a flow"
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy]: with 0 bandwidth

Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy]: done

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy [hunk 3]
```diff
@@ -29,11 +29,37 @@ class MetersSpec extends BaseSpecification {
...
+        def meterToDelete = northbound.getAllMeters(sw.dpId).meterEntries.find {
+            !defaultMeters.meterEntries*.meterId.contains(it.meterId)
+        }.meterId
+        def deleteResult = northbound.deleteMeter(sw.dpId, meterToDelete)
+
+        then: "Delete operation should be successful"
+        deleteResult.deleted
+
+        and: "Delete the flow"
+        northbound.deleteFlow(flow.id)
```
[the remark this answers is not in the document]
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy]: Yes, we should. Thanks.

**Pull request merged**